addons/60fps/addon.json

```diff
@@ -1,10 +1,15 @@
 {
-  "name": "60FPS project player mode",
-  "description": "Alt+Click the green flag to toggle 60FPS.",
+  "name": "Custom project framerate",
+  "description": "Alt+Click the green flag to toggle between Scratch's default 30FPS and a custom framerate.",
   "info": [
+    {
+      "type": "info",
+      "text": "The framerate (FPS) determines how many blocks are run per second. For example, running at 60FPS doubles the execution speed.",
+      "id": "fpsExplanation"
+    },
```

Comment:
This note is probably not important when we already warn the user in the note below this one that projects may run faster.

```suggestion

```

To comment on this explanation, it is a little too simplified and needs some extra details, IMO. Changing the framerate mainly just changes how fast the screen refreshes (and users already know that). Raising it gets us smoother animations but doing so has the _side effect_ of changing how fast projects run because Scratch automatically iterates certain code each frame, which is stated in the other notice below this one.

The framerate also does not have an exact impact on how many blocks run in a given time period; there are _specific_ things that Scratch iterates or evaluates once per frame.

Maybe if we ever make a Scratch Addons wiki or user guide we'll get deeper into this. 🙂

Reply:
> Maybe if we ever make a Scratch Addons wiki or user guide we'll get deeper into this. 🙂

The TurboWarp docs already have an article about that. Your suggestion makes sense, I wrote this PR at 10 pm :)